src/app/components/common/node-editor/editor.ts

```diff
@@ -550,4 +550,12 @@ async function removeOldConnection(nodeId) {
   for (let c of outputConnections) {
     await editor.removeConnection(c);
   }
+
+}
+
+export function applyContentReordering(nodeId: string) {
+  let view = area.nodeViews.get(nodeId);
+  let { content } = area.area;
+  // Bring selected node in front of other nodes for better visual clarity
+  content.reorder(view.element, null);
```

Comment:
I believe, node should be moved to the front after clicking the node (anywhere) also (along with the existing behaviour)

Reply:
I agree!

Sort of bring to front, on click in area of a specific node.